content/log-files.md

```diff
@@ -19,26 +19,38 @@ tableOfContents: true
 
-Among the information collected by the log files includes:
+The information collected by the log files includes but is not limited to:
 
 * The installed operating system and active kernel version
 * A listing of the filesystems connected to your system, and the space used
 * Information about your firmware
 * A list of disks connected to your system, as well as connected USB devices
-* Sensors information showing temperatures and fanspeeds
+* Sensors information showing temperatures and fan-speeds
 * Comprehensive output from `syslog` and `journalctl`
 
+## Generating Log Files in Settings (Pop!_OS 21.10+)
+
+Starting in Pop!\_OS 21.10, logs are no longer generated through the <u>System76 Driver</u> application. Instead, a "Support" section has been added to the <u>Settings</u> application. Clicking the `Create Log Files` button will prompt you for your admin password, and then generate and save a file called `system76-logs.tgz`. This file is saved in your Home directory (`/home/username`).
+
+![Settings-Logs](/images/system76-driver/logs-in-settings.png)
+
 ## Generating Log Files Using the System76 Driver Application
 
-The System76 Driver can be opened by clicking the <u>Activities</u> button in the top left (or pressing the Ubuntu or Pop key), then searching for 'System76.' Among the applications listed should be the <u>System76 Driver</u> application. Go ahead and click the icon to launch the application, and enter your password when prompted to do so.
+The System76 Driver can be opened by clicking the `Activities` or `Applications` button in the top left (or pressing the <kbd>SUPER</kbd>, then searching for "System76." Among the applications listed should be the <u>System76 Driver</u> application. Click the icon to launch the application, and enter your password when prompted to do so.
 
-Next click on the button outlined in red in the <u>System76 Driver</u> application and there will be a file called `system76-logs.tgz` placed in your Home directory (/home/username)
+Next click on the button outlined in red in the <u>System76 Driver</u> application and a file called `system76-logs.tgz` will be created and placed in your Home directory (`/home/username`).
 
 ![CreateLogFiles](/images/system76-driver/CreateLogFiles.png)
 
 ### Manually Generating Log Files
 
-If for some reason you are unable to access the <u>System76 Driver</u> application, you can manually create a set of logs files by running a series of commands via the Terminal application. To do this, click <u>Activities</u> in the top left of your system, search for <u>Terminal</u>, then open the application that appears.
+If for some reason you are unable to access the <u>System76 Driver</u> application, you can manually create a set of logs files by running a series of commands via the <u>Terminal</u> application. To do this, click `Activities`/`Applications` in the top left of your system, search for "Terminal", then open the application that appears.
```

Comment:
Should it be "log files" instead of "logs files?"

Reply:
Ha, yes. It should be either "logs" or "log files."